ai_edge_torch/generative/examples/stable_diffusion/README.md

```diff
@@ -2,13 +2,33 @@
 This example shows how to use the Edge Generative API to convert a PyTorch Stable Diffusion v1.5 model to TFLite model, and run the image generation inference.
 
 ## Convert PyTorch to TFLite model
-1. Download PyTorch stable diffusion model [stable-diffusion-pytorch](https://github.com/kjsman/stable-diffusion-pytorch)
-1. Unzip the downloaded model weight into `$HOME/Downloads/stable_diffusion_data`
-1. Run `convert_to_tflite.py`. This will convert the PyTorch models into TFLite models. The stable diffusion model has four components: CLIP (text embedding), encoder, diffusion and decoder models. Each component is converted to a single TFLite model file.
+The example provides source checkpoints mapping, from original HuggingFace repo to reauthored PyTorch model.
+
+### SafeTensors model checkpoints from original HuggingFace repo
+1. Clone original HuggingFace stable diffusion repo [runwayml/stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5)
+2. Run `convert_to_tflite.py` and use `v1-5-pruned-emaonly.safetensors` as the source checkpoints for the conversion script. The stable diffusion model has four components: CLIP (text embedding), encoder, diffusion and decoder models. Each component is converted to a single TFLite model file. Notice that the conversion of encoder model is not supported yet.
```

Comment:
curious why we are using this checkpoint? is this the standard SD 1.5 HF checkpoint?